### ISSUE TITLE
Add option to change how files are sorted internally

### ISSUE THREAD

**Issue by jurplel** [jurplel/qView#121]
Change how they would be ordered when using the next and previous functions

**Comment by jeep70** [comment]
Honestly I can't see anybody actually using this...

**Comment by neoasket** [comment]
qView sorts files in a lexical order at the moment
```
image1.jpg
image10.jpg
image2.jpg
```
could you add an option to sort in a natural order, please?
```
image1.jpg
image2.jpg
image10.jpg
```

**Comment by jurplel** [comment]
Hi, I appreciate the suggestion, but I believe this is already implemented for the next release (cfc85d96ec002f953eadbceb9aa20b95785bb006)

**Comment by Hrxn** [comment]
Another suggestion: Randomized order (shuffle), what do you think?
